addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -126,16 +178,23 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
         overflow: hidden;
         font-size: $font-size-sm;
         margin-bottom: map-get($spacers, 1);
+        opacity: 0;
         transform: scale(1, 0);
         transition: all ease 0.3s;
     }
     .o_wsale_product_btn {
-        @include o-position-absolute(auto, 0, 100%, 0);
+        @include o-position-absolute(auto, auto, 100%, 0);
         padding-bottom: map-get($spacers, 1);
+        z-index: 2;
 
         .btn {
+            width: 46px;
+            padding: 0.375rem 0.75rem; // prevent default Bootstrap padding from being modified by theme variables
```

Comment:
Feels strange that the settings of the editor does not influence this, no?

Reply:
Its because the selector is stronger than just .btn

Reply:
I understood that, what I meant is that it should ;) Someone which has small buttons everywhere does not want big buttons which he cannot change on its ecommerce... but let's say we can change that post freeze